### PR TITLE
DUPLO-13027: update documentation duplocloud_aws_ssm_parameter move value from optional to required

### PR DESCRIPTION
**Pull request by cbseirani-duplo**

### CODE REVIEW

**Comment by zafarabbas** [comment]
Task linked: [DUPLO-13027 TF: Doc: SSM Parameter: Value is not optional as mentioned in document](https://app.clickup.com/t/8655600/DUPLO-13027)

File: docs/resources/aws_ssm_parameter.md [hunk 1]
```diff
@@ -34,14 +34,14 @@ resource "duplocloud_aws_ssm_parameter" "ssm_param" {
...
 - `type` (String) The type of the SSM parameter. Valid values are `String`, `StringList`, and `SecureString`.
+- `value` (String) The value of the SSM parameter.
 
 ### Optional
 
 - `allowed_pattern` (String)
 - `description` (String) The description of the SSM parameter.
 - `key_id` (String)
 - `timeouts` (Block, Optional) (see [below for nested schema](#nestedblock--timeouts))
-- `value` (String) The value of the SSM parameter.
```
Comment (ddiaz-duplo) [inline docs/resources/aws_ssm_parameter.md:44]: We generally don't modify the markup files directly. 
Make the change to the resource itself https://github.com/duplocloud/terraform-provider-duplocloud/blob/develop/duplocloud/resource_duplo_aws_ssm_parameter.go#L42
then run:
```sh
make doc
```
To generate the updated `aws_ssm_paramter.md` file

**Review by ddiaz-duplo — CHANGES_REQUESTED** [review]

**Review by joek-duplo — CHANGES_REQUESTED** [review]
See David's change request.

File: docs/resources/aws_ssm_parameter.md (hunk 1 above)
Comment (cbseirani-duplo) [reply docs/resources/aws_ssm_parameter.md:44]: tyty

**Pull request closed without merging**